content/en/logs/workspaces/sql_reference.md

```diff
@@ -0,0 +1,58 @@
+---
+title: SQL Reference
+further_reading:
+- link: "/logs/workspaces/"
+  tag: "Documentation"
+  text: "Learn more about Log Workspaces"
+---
+
+## Overview
+
+SQL functions are built-in operations in SQL (Structured Query Language) used for tasks like calculations, data manipulation, or formatting. This documentation is a reference for SQL support available in Logs Workspaces. It details:
```

Comment:
Nit: I'm not sure whether we should start this guide with a note about explaining SQL functions since this is a reference guide on SQL support as a whole.